app/helpers/finance_helper.rb

```diff
@@ -24,27 +10,14 @@ def float_to_percentage(number)
   end
 
   def total_payment(breakdown)
-    service_fee = breakdown[:service_fees].map { |params| params[:monthly] }.inject(&:+)
-    output_payment = breakdown[:output_payments].map { |params| params[:subtotal] }.inject(&:+)
-    other_fees = breakdown[:other_fees].values.map { |other_fee| other_fee[:subtotal] }.inject(&:+)
```

Comment:
I'm not a fan of this method either and think it should be removed in a future change. I think we want to be asking the breakdown (or other object) for the total payment rather than reach in summing all these values